.github/workflows/docs.yml

```diff
@@ -17,25 +17,27 @@ jobs:
       uses: actions/checkout@v2.3.4
       with:
         persist-credentials: false
-        
+
     - name: Install latest beta
       uses: actions-rs/toolchain@v1
       with:
         toolchain: beta
         components: rust-docs
         override: true
 
+    - uses: Swatinem/rust-cache@v1
+
     - name: Install mdbook
       run: |
         cd book
         curl -L https://github.com/rust-lang/mdBook/releases/download/v0.4.5/mdbook-v0.4.5-x86_64-unknown-linux-gnu.tar.gz | tar xz
         # Add the book directory to the $PATH
         echo "$GITHUB_WORKSPACE/book" >> $GITHUB_PATH
-    
+
```

Comment:
FYI this will conflict with #2290

Reply:
This will defer to that one, if we merge this at all